config/_default/config.toml

```
@@ -1,4 +1,4 @@
-baseurl = "https://edu.chainguard.dev/"
+baseUrl = "https://edu.chainguard.dev/"
```

```suggestion
baseURL = "https://edu.chainguard.dev/"
```

On Hugo config I am seeing this